feature-toggles/feature-toggles-api/src/main/java/com/duckduckgo/feature/toggles/api/FeatureToggles.kt

```diff
@@ -231,25 +231,28 @@ internal class ToggleImpl constructor(
     }
 
     override fun isEnabled(): Boolean {
-        fun evaluateLocalEnable(state: State): Boolean {
+        fun evaluateLocalEnable(state: State, isExperiment: Boolean): Boolean {
+            // variants are only considered for Experiment feature flags
+            val isVariantTreated = if (isExperiment) state.isVariantTreated(appVariantProvider.invoke()) else true
+
             return state.enable &&
-                state.isVariantTreated(appVariantProvider.invoke()) &&
+                isVariantTreated &&
                 appVersionProvider.invoke() >= (state.minSupportedVersion ?: 0)
         }
         // check if it should always be enabled for internal builds
         if (isInternalAlwaysEnabled && flavorNameProvider.invoke().lowercase() == "internal") {
             return true
         }
         // If there's not assigned variant yet and feature forces default variant, set default variant
-        if (appVariantProvider.invoke() == null && forcesDefaultVariant) {
+        if (appVariantProvider.invoke() == null && isExperiment) {
```

Comment:
I think this check would be `!isExperiment`? We want to default variant to null if currentVarianKey is null or if the feature toggle doesn't have the @Experiment annotation